docs/source/getting_started/getting_started.md

```diff
@@ -2,17 +2,32 @@
 
 Here we take a high-level look at how the system works. The API is divided into two parts: the back office and the front office.
 
-The back office system can be used to ingest your content into the system. Every piece of content you ingest is managed as one document within the Xayn system.
+The back office can be used to ingest your content into the system. Every piece of content you ingest is managed as one document within the Xayn system.
 
-A document is comprised of one or more snippets. Each snippet represents a piece of the content of the document. In the simplest case, you only have one snippet per document in such a case you can mentally replace snippet with document in most of the remaining documentation. Through as there are limits on
-how long a snippet can be, we provide functionality to either summarize the content or split the content of a document into multiple snippets. In many places both a document id and the id of a specific snippet can be used.
+The front office is used for all communication between a client form a user. The front office mainly provides following use cases:
 
-During ingestion, the system creates a mathematical representation of each snippet which is used to match the snippets to the user's interests and searches.
+- semantic search, i.e. trying to find documents which are semantically similar to
+    - ... a text query
+    - ... a document
+- personalized recommendations of documents for a  user
+    - this is based on semantic representations of the users interest
+    - this interests can either be aggregated on out system
+    - or provided ad-hoc so that we never store user specific information
+
+Based on this many other use cases can be implemented. For example the personalization API can be used to implement a 'for you' section and the semantic search based on a document can be used to provide a 'more like this' section or add 'more like this' functionality to a 'for you' section. Another common use-case is to use the document based semantic search to refine an existing search.
+
+![architectural overview](./architecture_overview.png)
+
+Before the search or personalization APIs can be used documents need to be ingested into our search system. In the simplest case each document will produce exactly one search target based on its content.
+Through in more complex cases it can produce multiple search target and/or pre-process the content in some way. In the first part of this documentation we will only cover the case where a document directly maps to a single snippet. The following parts will cover more advanced use-cases.
```

Comment:
I changed it in a way which I think is better and completely removed the term search target.

Through I personally would prefer if we could only use the term search target instead of snippet everywhere through the documentation as it's in my opinion easier to understand. The reason I did not do so is because we have the terminology of snippets hard coded in our public API so that would be quite confusing to use.